Snake/DatabaseLayer/Database.cs

```diff
@@ -1,36 +1,74 @@
-﻿using System.IO;
+﻿using System.Data;
+using System.Data.SqlClient;
+using System.Collections.Generic;
 
 namespace DatabaseLayer
 {
     public class Database
     {
+        private const int MaxNoOfScores = 5;
+        private SqlConnection sqlConnection;
+        private SqlCommand cmd;
+        private SqlDataReader sqlDataReader;
+        private string mazeLevel;
+
+        public Database(string mazeLevel)
+        {
+            sqlConnection = new SqlConnection("Server= localhost; Database= SnakeHighScores; Integrated Security=True;");
+            cmd = new SqlCommand();
+            cmd.CommandType = CommandType.Text;
+            cmd.Connection = sqlConnection;
+            this.mazeLevel = mazeLevel;
+        }
+
         const string FILENAME = "highscores.txt";
 
-        public int getHighSCore()
+        public List<HighScore> getHighSCore()
         {
-            int highScore;
+            cmd.CommandText = "SELECT * FROM HighScores where GameType = '" + mazeLevel + "'";

```

Comment:
probably fine for this app but look up sql injection. it is a security vulnerability. one of the mitigations is to avoid building up sql strings like this